extra/liquidexport

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""liquidexport – host a metrics HTTP endpoint with Prometheus formatted data from liquidctl
+
+This is an experimental script that collects stats from liquidctl and exposes them as a http://localhost:8098/metrics
+endpoint in the Prometheus text format.
+See: https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example
+
+Example metric with labels:
+# HELP liquidctl liquidctl exported metrics
+# TYPE liquidctl gauge
+liquidctl{device="NZXT Kraken X (X42, X52, X62 or X72)",sensor="liquid_temperature",unit="°C"} 33.6
+
+Usage:
+  liquidexport [options]
+  liquidexport --help
+  liquidexport --version
+
+Options:
+  --interval <seconds>    Update interval in seconds [default: 2]
+  --hid <module>          Override API for USB HIDs: usb, hid or hidraw
+  --legacy-690lc          Use Asetek 690LC in legacy mode (old Krakens)
+  --match <name>          Filter devices by name
+  --vendor <id>           Filter devices by vendor id
+  --product <id>          Filter devices by product id
+  --release <number>      Filter devices by release number
+  --serial <number>       Filter devices by serial number
+  --bus <bus>             Filter devices by bus
+  --address <address>     Filter devices by address in bus
+  --usb-port <port>       Filter devices by USB port in bus
+  --pick <number>         Pick among many results for a given filter
+  --port <number>         Port for the HTTP /metrics endpoint
```

Comment:
Having both `--usb-port` and `--port` may be confusing.

It may be reasonable to remove device filtering altogether (`--legacy-690lc` is still necessary because it affects driver behavior).

Or maybe rename the specific option to `--server-port`?

Reply:
I removed the device filtering, but copied over some arg validation to keep `--legacy-690lc` working and flowing into `find_liquidctl_devices(**opts)` which is where I assume it is used by the drivers.

I also renamed the other key to `--server-port` to differentiate it from the USB ports in other scripts.